src/index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+    </head>
+<body>
+    <h1 class="header">Leaderboard</h1>
+<main class=" justify-content main">
+    <section class="display-section">
+        <div class="flex justify-content scores">
+            <h2>Recent scores</h2>
+            <button type="reset">Refresh</button>
+        </div>
+        <ul class="display">
+         <li>
+             <ul class="flex">
+                <li>Name<span>:</span></li>
+                <li>100</li>
+             </ul>
+             <ul class="flex">
+                <li>Name<span>:</span></li>
+                <li>20</li>
+             </ul>
+             <ul class="flex">
+                <li>Name<span>:</span></li>
+                <li>50</li>
+             </ul>
+             <ul class="flex">
+                <li>Name<span>:</span></li>
+                <li>78</li>
+             </ul>
+             <ul class="flex">
+                <li>Name<span>:</span></li>
+                <li>125</li>
+             </ul>
+             <ul class="flex">
+                <li>Name<span>:</span></li>
+                <li>77</li>
+             </ul>
+             <ul class="flex">
+                <li>Name<span>:</span></li>
+                <li>42</li>
+             </ul>
+         </li>
+        </ul>
+    </section>
+    <section class="form-section">
+       <h2 class="form-section-header">Add your score</h2>
+       <form>
+           <label><input type="text" placeholder="Your name"></label><br>
+           <label><input type="number" placeholder="Your score"></label><br>
+      <button type="submit">Submit</button>
+        </form>
+    </section>
+</main>
+<footer class="footer"></footer>
```

Comment:
- [ ] Kindly move all this code into `index.js` file. The goal is to get you comfortable with working with proper ES6 styling such as using `arrow functions`, proper usage of `const`, and `let` keywords.
- [ ] The index.html body should contain only the root element which is a link to the index.js file